Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -22,13 +22,21 @@ def install
     (buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve").install Dir["{*,.git,.gitignore}"]
 
     ENV["GOPATH"] = "#{buildpath}/gopath"
-
-    cd buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve" do
+    build_root = buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve"
+    cd build_root do
       if build.head?
         git_hash = `git rev-parse --short HEAD --quiet`.chomp
-        git_hash = " HEAD-#{git_hash}"
+        git_hash = "HEAD-#{git_hash}"
+        ENV["CGO_LDFLAGS"] = "#{build_root}/vendor/build/lib9p/lib9p.a -L#{build_root}/vendor/lib9p"
+        ENV["CGO_CFLAGS"] = "-I#{build_root}/vendor/lib9p"
+        mkdir "vendor/build"
```

Comment:
I fix `make` job in https://github.com/zchee/docker-machine-driver-xhyve/pull/106
`mkdir` line is no longer required. It creates build directory automatically even Homebrew.
I tested local `brew` command.

@DomT4 WDYT?